### ISSUE TITLE
Release 0.0.104 broke 'npm test [package]'

### ISSUE THREAD

**Issue by plumdog** [microsoft/DefinitelyTyped-tools#402]
Clone https://github.com/DefinitelyTyped/DefinitelyTyped, cd into it and `npm i`, then:
```bash
$ npm i @definitelytyped/dtslint@0.0.103 && npm test through
...

> definitely-typed@0.0.3 test
> dtslint types "through"

$ npm i @definitelytyped/dtslint@0.0.104 && npm test through
...

> definitely-typed@0.0.3 test
> dtslint types "through"

Error: ENOENT: no such file or directory, open '../notNeededPackages.json'
```

I haven't tried to dig into why. It might be that I'm not understanding something, but I think I'm following the docs at https://github.com/DefinitelyTyped/DefinitelyTyped#running-tests.

**Comment by plumdog** [comment]
https://github.com/microsoft/DefinitelyTyped-tools/compare/v0.0.103...v0.0.104#diff-072a98b65a763133fbb40f269b51c6d57f24ae262e81cad439304be63de7375eR141 :thinking:

**Comment by plumdog** [comment]
Fixed in #405, in release 0.0.108
